include/ofi_abi.h

```diff
@@ -111,7 +111,7 @@ extern "C" {
  * name appended with the ABI version that it is compatible with.
  */
 
-#define CURRENT_ABI "FABRIC_1.7"
+#define CURRENT_ABI "FABRIC_1.8"
```

Comment:
It's not clear to me why https://github.com/ofiwg/libfabric/pull/10279/commits/7a26b036d3f944dadbc6439ad3bd7eaa81bf01ed didn't bump the ABI, I am asking intel offline